documents/invoice/model_test.go

```diff
@@ -44,6 +44,7 @@ func TestMain(m *testing.M) {
 		&testlogging.TestLoggingBootstrapper{},
 		&config.Bootstrapper{},
 		&storage.Bootstrapper{},
+		&identity.Bootstrapper{},
```

Comment:
Can we not have pointer receiver for this?

Reply:
same comment as above, but lets discuss, agree and change this for all bootstrappers in a single pr later.